decidim-core/app/packs/src/decidim/map/provider/here.js

```diff
@@ -8,6 +9,8 @@ import * as L from "leaflet"
  */
 $(() => {
   $("[data-decidim-map]").on("configure.decidim", (_ev, map, mapConfig) => {
-    L.tileLayer.here(mapConfig.tileLayer).addTo(map);
+    let tileLayer = mapConfig.tileLayer;
+    tileLayer.language = hereMapLanguageCode($("html").attr("lang"));
```

Comment:
I would prefer to do this at the backend at the HERE Maps builder class which has access to the view context:
https://github.com/decidim/decidim/blob/f8a9e619f32df65fc8afb5a435cb2c27d9cf856d/decidim-core/lib/decidim/map/provider/dynamic_map/here.rb#L41

You can add the following methods there to provide the tile layer options to the view:

```ruby
            protected

            def options
              super.tap do |opts|
                opts[:tile_layer].merge!(
                  language: convert_locale(template.controller.current_locale)
                )
              end
            end

            def convert_locale(locale)
              # TODO: Implement the locale conversion here
              "eng"
            end
```

Note that you need to access the `current_locale` through the controller because the builders are also used within the cells which may not always have the `current_locale` method available to them.

Also, could you then add tests for this added functionality at:
https://github.com/decidim/decidim/blob/f8a9e619f32df65fc8afb5a435cb2c27d9cf856d/decidim-core/spec/lib/map/provider/dynamic_map/here_spec.rb#L9